rotkehlchen/tasks/manager.py

```diff
@@ -653,11 +653,9 @@ def _maybe_run_events_processing(self) -> Optional[list[gevent.Greenlet]]:
         )]
 
     def _maybe_update_yearn_vaults(self) -> Optional[list[gevent.Greenlet]]:
-        if has_tracked_addresses(
```

Comment:
so both for yearn vaults and ilk cache we always got in this code, even if user was tracking no ethereum addresses.